test/addons/esm/test-esm.mjs

```diff
@@ -0,0 +1,50 @@
+// eslint-disable-next-line node-core/require-common-first
+import { buildType } from '../../common/index.mjs';
+import assert from 'node:assert';
+import { createRequire } from 'node:module';
+
+const require = createRequire(import.meta.url);
+
+export async function run() {
+  // binding.node
+  {
```

Comment:
Please use `describe` + `it`

Reply:
I personally find `describe` + `it` to be worse UX due to the reasons stated in https://github.com/nodejs/node/pull/56027#issuecomment-2504323462, unless there is a way to silence the passing tests and make the console logs from the failed test cases appear at the end together with the errors..